<launch>
    <arg name="config_path" default = "$(find air_vo)/configs/configs_realsense.yaml" />
    <arg name="dataroot" default = "/media/xukuan/something2/ubuntu_files/data/AirSLAM/realsense_meeting_room/euroc_style" />
    <arg name="camera_config_path" default = "$(find air_vo)/configs/realsense_848_480.yaml" />
    <arg name="model_dir" default = "$(find air_vo)/output" />
    <arg name="saving_dir" default = "$(find air_vo)/debug" />

    <node name="air_vo" pkg="air_vo" type="air_vo" output="screen">
      <param name="config_path" type="string" value="$(arg config_path)" />
      <param name="dataroot" type="string" value="$(arg dataroot)" />
      <param name="camera_config_path" type="string" value="$(arg camera_config_path)" />
      <param name="model_dir" type="string" value="$(arg model_dir)" />
      <param name="saving_dir" type="string" value="$(arg saving_dir)" />
    </node>
</launch>		